src/reanimated2/hook/useAnimatedStyle.ts

```diff
@@ -510,20 +512,27 @@ For more, see the docs: \`https://docs.swmansion.com/react-native-reanimated/doc
     return () => {
       stopMapper(mapperId);
     };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, dependencies);
 
   useEffect(() => {
-    animationsActive.value = true;
+    areAnimationsActive.value = true;
     return () => {
-      animationsActive.value = false;
+      areAnimationsActive.value = false;
     };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
   checkSharedValueUsage(initial.value);
 
   if (isJest()) {
-    return { viewDescriptors, initial, viewsRef, animatedStyle };
+    return {
+      viewDescriptors,
+      initial,
+      maybeViewsRef,
```

Comment:
Do we really want to rename the key here?
```suggestion
      viewsRef: maybeViewsRef,
```

Reply:
Seems like a mistake.